### PR TITLE
Update prism to v1.21.0

### PR DESCRIPTION
**Pull request by XhmikosR**
Preview: https://elastic-heyrovsky-9fc9ac.netlify.app/

### CODE REVIEW

**Comment by XhmikosR** [comment]
BTW @bryanbraun have you thought about deploying to the gh-pages branch only on tags or manually? Right now, there's a chance that docs are broken if one does not run `npm run build`. We can automate this with GitHub Actions easily.

File: docs/index.html [hunk 1]
```diff
@@ -944,6 +944,6 @@ <h3>Tooltip</h3>
     var hovEx5 = new AnchorJS({ icon: 'Permalink' });
     hovEx5.add('.hover-examples .example:nth-child(5) h3');
   </script>
-  <script src="https://cdn.jsdelivr.net/prism/1.5.1/prism.js" async></script>
+  <script src="https://cdn.jsdelivr.net/npm/prismjs@1.21.0/prism.min.js" async></script>
```
Comment (bryanbraun) [inline docs/index.html:947]: Ooo, getting the minified version is a nice improvement. 😄

**Comment by bryanbraun** [comment]
> BTW @bryanbraun have you thought about deploying to the gh-pages branch only on tags or manually? Right now, there's a chance that docs are broken if one does not run `npm run build`. We can automate this with GitHub Actions easily.

Yeah the "Copy to docs" setup is a little awkward. I'm inclined to not refine this too much right now, because I suspect that some of it will change after I work on https://github.com/bryanbraun/anchorjs/issues/109 .

One option is to deploy the whole repo to github-pages instead of just the `/docs` folder. That would allow me to link the JS file directly, so the docs would reference the `anchor.js` on `master`. I did something like that for https://github.com/bryanbraun/checkboxland.

Github Actions is pretty nice too, though, so lots of options. 😄

**Comment by XhmikosR** [comment]
I think the best approach would be to deploy with GitHub Actions. That way, you can make sure the live site is indeed the latest master code, assuming you want that and not the latest tag (we can go either way). For example, we deploy our blog from the main branch on https://github.com/twbs/blog/blob/main/.github/workflows/ci.yml

Anyway, let me know if you'd like me to submit a PR for this. You will just need to change the deployment options :)

**Comment by bryanbraun** [comment]
@XhmikosR, thanks for the example with twbs. I'll keep thinking about the deploy thing. I'll create an issue and ping you if I feel like it's time to go in that direction. 👍

**Pull request merged**